### PR TITLE
Fix pylint duplicate code error

### PR DESCRIPTION
**Pull request by JenySadadia**
Fixes https://github.com/kernelci/kernelci-api/issues/113
Created on top of https://github.com/kernelci/kernelci-api/pull/112

### CODE REVIEW

File: test/test_node_handler.py [hunk 1]
```diff
@@ -3,76 +3,25 @@
...
+"""Unit test functions for KernelCI API node handler"""
 
-
-@pytest.fixture()
-def mock_db_find_by_id(mocker):
-    async_mock = AsyncMock()
-    mocker.patch('api.db.Database.find_by_id',
-                 side_effect=async_mock)
-    return async_mock
+import json
```
Comment (mgalka) [inline test/test_node_handler.py:10]: Maybe we could use the refactoring opportunity and sort imports alphabetically?

File: test/conftest.py [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+
+"""pytest fixtures for KernelCI API"""
+
+from unittest.mock import AsyncMock
+import fakeredis.aioredis
+import pytest
+from fastapi.testclient import TestClient
+from api.main import app
+from api.models import User
+from api.pubsub import PubSub
```
Comment (mgalka) [inline test/conftest.py:16]: Please, sort imports alphabetically

**Review by mgalka — CHANGES_REQUESTED** [review]
Some cosmetic changes only.

**Review by mgalka — COMMENTED** [review]
Let's address the issues from https://github.com/kernelci/kernelci-api/pull/112 and then we can merge this one

**Comment by JenySadadia** [comment]
> Some cosmetic changes only.

Not able to see the suggested changes in the comment.
Maybe after rebasing comments disappeared.

File: test/conftest.py [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+    return async_mock
+
+
+@pytest.fixture
+def get_bearer_token():
+    """Returns bearer token to be used for Authentication"""
+    return "Bearer \
+            eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9. \
+            eyJzdWIiOiJib2IifQ.ci1smeJeuX779PptTkuaG1S \
+            Edkp5M1S1AgYvX8VdB20"
```
Comment (gctucker) [inline test/conftest.py:159]: Why is this a fixture and not a regular function?

Comment (JenySadadia) [reply test/conftest.py:159]: If I define it as a regular function, I will need to import it by `from test.conftest import get_bearer_token`.
In the case of fixture, it automatically gets imported by defining in conftest.py just like other fixtures.

Comment (gctucker) [reply test/conftest.py]: OK, but I don't see a good reason for not just importing the module and using it like any regular Python code.

Comment (gctucker) [reply test/conftest.py]: In fact, the string could also be defined directly in the module e.g. `conftest.BEARER_TOKEN`.

**Comment by JenySadadia** [comment]
Added conftest.BEARER_TOKEN.
Needed to add import statement for it before TestClient to resolve below pylint error:
```
************* Module test.test_listen_handler
test/test_listen_handler.py:16:0: C0411: standard import "from test.conftest import BEARER_TOKEN" should be placed before "from fastapi.testclient import TestClient" (wrong-import-order)
************* Module test.test_me_handler
test/test_me_handler.py:13:0: C0411: standard import "from test.conftest import BEARER_TOKEN" should be placed before "from fastapi.testclient import TestClient" (wrong-import-order)
************* Module test.test_node_handler
test/test_node_handler.py:16:0: C0411: standard import "from test.conftest import BEARER_TOKEN" should be placed before "from fastapi.testclient import TestClient" (wrong-import-order)
************* Module test.test_subscribe_handler
test/test_subscribe_handler.py:14:0: C0411: standard import "from test.conftest import BEARER_TOKEN" should be placed before "from fastapi.testclient import TestClient" (wrong-import-order)
************* Module test.test_unsubscribe_handler
test/test_unsubscribe_handler.py:13:0: C0411: standard import "from test.conftest import BEARER_TOKEN" should be placed before "from fastapi.testclient import TestClient" (wrong-import-order)

-------------------------------------------------------------------
Your code has been rated at 9.78/10 (previous run: 10.00/10, -0.22)
```

**Review by mgalka — APPROVED** [review]

**Pull request merged**